pyproject.toml

```
@@ -25,6 +25,9 @@ pydub = ">=0.25.1,<1.0.0"
 jiwer = "^3.0.1"
 wandb = ">=0.15.3,<1.0.0"
 accelerate = ">=0.19.0,<1.0.0"
+openpyxl = "^3.1.2"
+pycountry = "^22.3.5"
+wave = "^0.0.2"
```

We want to automatically update the dependencies through both patch and minor versions. If we write `wave=^0.0.2` then only patch versions will be updated, so we would never update to, e.g., version 0.1.0.

```suggestion
wave = ">=0.0.2,<1.0.0"
```